workflow-templates/label-on-comment.yml

```diff
@@ -0,0 +1,20 @@
+# This workflow runs when a comment is made on the ticket
+# If the comment starts with "label: " it tries to apply
+# the label indicated in rest of comment.
+# If the comment starts with "remove label: ", it tries
+# to remove the indicated label.
+# Note: Labels are allowed to have spaces and this script does
+# not parse spaces (as often a space is legitimate), so the command
+# "label: really long lots of words label" will apply the
+# label "really long lots of words label"
+
+name: Allows for the adding and removing of labels via comment
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  add_label:
```

Comment:
I would call this job "process_comment" or "add_delete_labels" or something. 

Here's the way this works:

- You're defining ONE job here that references ALL the jobs in the file below
- If that file contains multiple jobs, all will be run

Thus, the job you define to _call_ this file should be descriptive of all jobs that are being run.

Based on this comment I actually think we should change the name of the workflow to `add-remove-label-on-comment`. I named this file when only having the "add" action, now it also has the "delete" action.

Reply:
Thank you for explaining this. I've updated all the workflow template with the job name `add_remove_labels`, changed the name of the workflow template `add-remove-label-on-comment.yml`, and updated the workflows in BTR WG and public engineering repos.

Reply:
One last thing, you also need to rename the source workflow too.

Reply:
Done!